iFixit/react-commerce
# Add initial product page template behind feature flag

This adds a product page template stub behind a feature flag, as we don't want to release this before it's ready. 
I'll be adding one by one the pieces that I've worked on last month, this is just to start developing with feature flags.

This implementation uses environment variables to control flags.

## QA

1. Open [prod-like Vercel preview](https://react-commerce-prod-git-add-product-page-template-ifixit.vercel.app/Products/test) (flag not enabled)
2. Verify that `/Products/test` returns a 404
3. Visit [staging Vercel preview](https://react-commerce-git-add-product-page-template-ifixit.vercel.app/Products/test) (flag is enabled)
4. Verify that you can see a product page stub at `/Products/test`
